index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Document</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="container">
+      <section class="c-nav-button">
+        <nav class="c-burger-menu">
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            width="74"
+            height="74"
+            viewBox="0 0 74 74"
+            class="burger-icon"
+          >
+            <g id="Burger-icon" transform="translate(-190 -596)">
+              <rect
+                id="Rectangle_1"
+                data-name="Rectangle 1"
+                width="74"
+                height="74"
+                rx="26"
+                transform="translate(190 596)"
+                fill="#3b3f54"
+              />
+              <line
+                id="Line_257"
+                data-name="Line 257"
+                x2="10"
+                transform="translate(230 625)"
+                fill="none"
+                stroke="#f3f4fb"
+                stroke-linecap="round"
+                stroke-width="3"
+              />
+              <line
+                id="Line_258"
+                data-name="Line 258"
+                x2="24"
+                transform="translate(216 635)"
+                fill="none"
+                stroke="#f3f4fb"
+                stroke-linecap="round"
+                stroke-width="3"
+              />
+              <line
+                id="Line_259"
+                data-name="Line 259"
+                x2="10"
+                transform="translate(216 645)"
+                fill="none"
+                stroke="#f3f4fb"
+                stroke-linecap="round"
+                stroke-width="3"
+              />
+            </g>
+          </svg>
+        </nav>
+      </section>
+      <div class="c-main">
+        <section class="c-header">
+          <h1 class="c-header__text u-text--h">Hello There</h1>
+          <p class="c-header__text u-text--p">
+            Welcome back to the workspace, we missed You!
+          </p>
+          <div class="c-search-bar">
+            <svg
+              id="search_1_"
+              data-name="search (1)"
```

Comment:
What's this for?

Reply:
Search button on search bar 
ID has been removed